### DISCUSSION TITLE
Window Grid Similar to Hammerspoon

### DISCUSSION THREAD

**Discussion by christianmagill** [rxhanson/Rectangle#610] [Ideas]
How about incorporating a window grid option similar to Hammerspoon.

This would give a lot of flexibility.

https://youtu.be/mZL1QejOQ5A?t=158

**Comment by rxhanson** [comment]
I don’t have plans to add this into Rectangle, mainly because I like keeping Rectangle simple. My take on custom sizes is built into my paid superset of Rectangle, Hookshot (https://hookshot.app). It’s the wrench and ruler tab in the Hookshot prefs window.

**Reply by davidbernard04** [reply]
Woah I too wasn't aware there was a paid version with more features like custom sizes! Just tried it and worked perfectly, will surely buy it.

PS: You should add a menu in Rectangle like "Custom Sizes" that will lead to Hookshot, so people would be aware of that. :)

**Comment by christianmagill** [comment]
Thanks, I wasn't aware of Hookshot, I'll check it out.
